src/win/fs-event.c

```diff
@@ -146,7 +155,8 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   int name_size, is_path_dir;
   DWORD attr, last_error;
   WCHAR* dir = NULL, *dir_to_watch, *pathw = NULL;
-  WCHAR short_path[MAX_PATH];
+  WCHAR short_path_buffer[MAX_PATH];
+  WCHAR* short_path = short_path_buffer;
```

Comment:
Declaration with assignment is used all over the place in libuv, so I guess it is ok the way it is?